stackstorm-nexus3/pack.yaml

```diff
@@ -0,0 +1,13 @@
+---
+ref: nexus3
```

Comment:
With a ref like this I would only ever expect it to work with Nexus Repository 3 and not Nexus Repository 2 or any future Nexus Repository 4 or Nexus Repository 15.

But that is fine. We can create a pack for future versions of Nexus Repository.

Reply:
Intention is to keep it fully supported with nexus3. Due to update in endpoints, it doesn't work with nexus2 and so might be true for future versions until they fully adopt REST for all resources.